<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QMainWindow" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cryotec Клиент</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>27</x>
      <y>31</y>
      <width>691</width>
      <height>571</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QTreeWidget" name="machines_tree">
     <property name="headerHidden">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
    <widget class="QWidget" name="">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tabReport">
         <attribute name="title">
          <string>Неисправности</string>
         </attribute>
         <widget class="QWidget" name="">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>0</y>
            <width>331</width>
            <height>441</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="create_report_button">
               <property name="text">
                <string>Создать</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableView" name="report_table"/>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="tabFix">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Страница</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="sync_button">
        <property name="text">
         <string>Синхронизировать изменения</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>sync_button</zorder>
   <zorder>tabWidget</zorder>
   <zorder>sync_button</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>892</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
